### PR TITLE
added verified badge to profile and removed/dropped smartContractAddress from db

### PR DESCRIPTION
**Pull request by PeerRich**
image

### CODE REVIEW

**Comment by jamiepine** [comment]
CleanShot 2022-02-05 at 12 09 02@2x
Build is failing because I modified the migration, however this was due to the migration failing to run the previous commit and Prisma does not apply the migration if it failed, so how did it run on the staging database? I don't have access so I can't inspect the logs. Unless someone can delete the record of the migration I ran(?) in staging I will roll back the commits.

**Comment by PeerRich** [comment]
> CleanShot 2022-02-05 at 12 09 02@2x
> 
> Build is failing because I modified the migration, however this was due to the migration failing to run the previous commit and Prisma does not apply the migration if it failed, so how did it run on the staging database? I don't have access so I can't inspect the logs. Unless someone can delete the record of the migration I ran(?) in staging I will roll back the migration edit. 

@zomars @baileypumfleet @emrysal can someone help? 🙏

File: pages/[user].tsx [hunk 1]
```diff
@@ -33,8 +34,8 @@ export default function User(props: inferSSRProps<typeof getServerSideProps>) {
   delete query.user; // So it doesn't display in the Link (and make tests fail)
 
   const nameOrUsername = user.name || user.username || "";
-
   const [evtsToVerify, setEvtsToVerify] = useState<EvtsToVerify>({});
+  const verified = user.verified;
```
Comment (baileypumfleet) [inline pages/[user].tsx:38]: Personally, I'd just directly use `user.verified` and skip this extra line of code

File: prisma/migrations/20220205135022_add_verified_column/migration.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `smartContractAddress` on the `EventType` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "EventType" DROP COLUMN IF EXISTS "smartContractAddress";
```
Comment (baileypumfleet) [inline prisma/migrations/20220205135022_add_verified_column/migration.sql:8]: Things like this should really be in a separate PR, because it's not clear that the removal of the `smartContractAddress` column would be found in the verified badge PR

File: pages/[user].tsx [hunk 3]
```diff
@@ -55,7 +56,8 @@ export default function User(props: inferSSRProps<typeof getServerSideProps>) {
                 alt={nameOrUsername}
               />
               <h1 className="mb-1 text-3xl font-bold font-cal text-neutral-900 dark:text-white">
-                {nameOrUsername}
+                {nameOrUsername}{" "}
```
Comment (baileypumfleet) [inline pages/[user].tsx:59]: Very small nitpick, but I'd render the space conditionally if verified is true, otherwise we have a random space.

Can be fixed by adding this to the line below:
```
verified && " " + <Verified...
```

**Review by pumfleet — APPROVED** [review]
Looks good, just have a few comments for consideration

File: prisma/migrations/20220205135022_add_verified_column/migration.sql (hunk 2 above)
Comment (PeerRich) [reply prisma/migrations/20220205135022_add_verified_column/migration.sql:8]: oh i dont even know where this came from. i really wanna get this merged, is this a hard requirement?

Comment (PeerRich) [reply prisma/migrations/20220205135022_add_verified_column/migration.sql:8]: added the title: added verified badge to profile and removed/dropped smartContractAddress from db #1710
